lib/metadata_placement.js

```diff
@@ -18,7 +18,7 @@ var EventEmitter = require('events').EventEmitter;
 
 var assert = require('assert-plus');
 var bignum = require('bignum');
-var boray = require('boray');
+var mdapi = require('buckets-api');
```

Comment:
Shouldn't this also be `require('buckets-mdapi')`?

Reply:
ug, sorry about that.